source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -1,5 +1,9 @@
-#include <Zivid/Zivid.h>
+/*
+Zividbenchmarks is a sample that will test the average speed of different operations on your computer. It will provide
```

Comment:
- Zivid**B**enchmark - capital B
- _the average speed_ - can we change this to _performance_? Emphesize that it is performance of the computer on using Zivid camera, not performance in general.
- _the mean and median_ - add _time_ - I would maybe even skip saying it is median and mean, time is sufficient I think. Maybe we add min, max, st. dev at some point, etc. no need to go into details in description.
- Let's try to use the same terms that we use in printing, e.g. _breakdown of the total capture time for single and HDR captures with and without filters_ instead of _single imaging, HDR and filtering._
We also measure _saving_ point clouds.

Reply:
Changed the wording and phrasing to match closer what we are doing in the code.